mix.exs

```diff
@@ -16,7 +16,8 @@ defmodule Exavier.MixProject do
 
   def application do
     [
-      extra_applications: [:logger]
+      extra_applications: [:logger],
+      env: [test_file_to_module_func: [Exavier, :test_file_to_module_func]]
```

Comment:
redundant?

Reply:
I don't know actually.. Will users have our configs from `config.exs`? If they will then this is redundant.